back/app/services/copies_updating_service.rb

```diff
@@ -0,0 +1,39 @@
+class CopiesUpdatingService
+  def update_custom_fields
+    scope = CustomField.where(resource_type: User.name)
+    %w[gender birthyear domicile education].each do |code|
+      scope.where(code: code).find_each do |record|
+        update_multiloc_value(record, :title_multiloc, "custom_fields.users.#{code}.title")
+      end
+    end
+  end
+
+  private
+
+  def update_multiloc_value(record, attr_name, copy_path)
+    old_multiloc_value = record.public_send(attr_name)
+    new_multiloc_value = latest_multiloc_value(copy_path)
+
+    old_multiloc_value.each do |(locale, value)|
+      english = locale.starts_with?('en')
+      value_translated = value != old_multiloc_value['en']
+
+      new_multiloc_value[locale] = value if english || value_translated
```

Comment:
Then it sets previous copies for English and other translated copies. So, basically, it updates only not translated copies + adds new locales.